src/layer/vector/Polyline.js

```diff
@@ -172,6 +172,21 @@ export var Polyline = Path.extend({
 		return this.redraw();
 	},
 
+	// @method setLastLatLng(latlng: LatLng, latlngs? LatLng[]): this
+	// Modifies the last point of the polyline to be the specified point.
+	// By default, adds to the first ring of the polyline in case of a
+	// multi-polyline, but can be overridden by passing a specific ring as a
+	// LatLng array (that you can earlier access with
+	// [`getLatLngs`](#polyline-getlatlngs)).
+	setLastLatLng: function (latlng, latlngs) {
+		latlngs = latlngs || this._defaultShape();
+		latlng = toLatLng(latlng);
+		latlngs.pop();
+		latlngs.push(latlng);
+		this._bounds.extend(latlng);
+		return this.redraw();
```

Comment:
...and `redraw()` will redraw the whole geometry anyway, which means any performance gains from this new method are gonna be negligible.

Reply:
I was hoping to gain performance from avoiding the instantiation + copying of a potentially long array on each mouse move. I'll try with the array copying and see if I run into any performance problems.